manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.2.3 $debug=QUSU.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: voidtools.Everything.Alpha
+PackageVersion: 1.5.0.1339a
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.voidtools.com/Everything-1.5.0.1339a.x64-Setup.exe
+  InstallerSha256: 76A78E1F23BD4430AC78E7B70B7738B0867D38D6E414C7C2FF843E0B307A85BA
+  ProductCode: Everything 1.5a
```

```suggestion
  ProductCode: Everything 1.5a
  AppsAndFeaturesEntries:
    - DisplayName: Everything 1.5.0.1339a (x64)
      DisplayVersion: 1.5.0.1339
```
To match Control Panel / Apps&Features